tests/test_packages.py

```diff
@@ -0,0 +1,162 @@
+from testfm.packages import Packages
+from testfm.log import logger
+
+
+def test_positive_foreman_maintain_packages_lock(setup_install_pkgs, ansible_module):
+    """Verify whether satellite related packages get locked
+
+    :id: d387d8be-10ad-4a62-aeff-3bc6a82e6bae
+
+    :setup:
+        1. foreman-maintain should be installed.
+
+    :steps:
+        1. Run foreman-maintain packages lock
+        2. Run foreman-maintain packages status
+        3. Run foreman-maintain packages is-locked
+        4. check 'satellite' is mentioned
+        in /etc/yum/pluginconf.d/versionlock.list
+        5. Run foreman-maintain packages unlock
+        6. Run foreman-maintain packages status
+        7. Run foreman-maintain packages is-locked
+        8. check 'satellite' is not mentioned
+        in /etc/yum/pluginconf.d/versionlock.list
+
+    :expectedresults: expected packages get locked and unlocked.
+
+    :CaseImportance: Critical
+    """
+    sat_pkgs = 0
+    contacted = ansible_module.command(Packages.lock())
+    for result in contacted.values():
+        logger.info(result['stdout'])
+        assert "FAIL" not in result['stdout']
+        assert result["rc"] == 0
+    contacted = ansible_module.command(Packages.status())
+    for result in contacted.values():
+        logger.info(result['stdout'])
+        assert 'Packages are locked.' in result['stdout']
+        assert "FAIL" not in result['stdout']
+        assert result["rc"] == 0
+    contacted = ansible_module.command(Packages.is_locked())
+    for result in contacted.values():
+        logger.info(result['stdout'])
+        assert 'Foreman related packages are locked' in result['stdout']
+        assert result["rc"] == 0
+    contacted = ansible_module.command("grep 'satellite' /etc/yum/pluginconf.d/versionlock.list")
+    for result in contacted.values():
+        logger.info(result['stdout'])
+        assert 'satellite' in result['stdout']
+    contacted = ansible_module.shell("yum repolist")
+    for result in contacted.values():
+        logger.info(result['stdout'])
+        sat_pkgs = contacted.values()[0]['stdout_lines'][-1][-3:]
```

Comment:
I want the count of total no. of packages available, but don't have any ansible module for that.

Reply:
> I want the count of total no. of packages available, but don't have any ansible module for that.

@ntkathole  because of it, I have hard coded this part. It'll be helpful if you have any other suggestion on this one.